ansible/roles/configure-batyam/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
CI should be in a different PR.
We want to use GitHub Actions instead of Travis CI.

Reply:
That was in the template, will remove